pyccel/ast/core.py

```diff
@@ -3371,8 +3380,68 @@ def attributes_as_dict(self):
             d_attributes[i.name] = i
         return d_attributes
 
-    # TODO add other attributes?
+    def add_new_attribute(self, attr):
+        """
+        Add new attributes to the current class.
+
+        A new attributes of the current ClassDef.
+
+        Parameters
+        ----------
+        attr : Varibale
+            The Variable that will be added.
+
+        Returns
+        -------
+        None
```

Comment:
You don't need to provide a `Returns` section if nothing is returned

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1484/commits/be10551c86b0a69ce02281606259134a26ed64a7.